### PR TITLE
es: Translate unstranlsated paragraphs

### PR DESCRIPTION
**Pull request by Graywolf9**
### Description

Traducción de fragmentos sin traducir en los artículos. Lo he hecho de 10 archivos ya que son por párrafo, para avanzar más y para que sea un número fácil de revisar. Si es necesario puedo separarlo.

Nota: Esto no es una actualización de contenido

### Related issues and pull requests

Relates to https://github.com/mdn/translated-content/issues/9638

### CODE REVIEW

**Review by JuanVqz — APPROVED** [review]
🏅 Thanks! looks good

File: files/es/learn/common_questions/web_mechanics/what_are_hyperlinks/index.md [hunk 1]
```diff
@@ -12,12 +12,12 @@ En este artículo, repasaremos qué son los hipervínculos y por qué son import
       <th scope="row">Prerrequisitos:</th>
       <td>
         Debes saber
-        <a href="/en-US/Learn/How_the_Internet_works">how the Internet works</a>
+        <a href="/es/Learn/How_the_Internet_works">como funciona internet</a>.
```
Comment (JuanVqz) [inline files/es/learn/common_questions/web_mechanics/what_are_hyperlinks/index.md:15]: Should we add the question marks?

**Review by JuanVqz — APPROVED** [review]
In general, this is looking good!

File: files/es/learn/common_questions/web_mechanics/what_are_hyperlinks/index.md (hunk 1 above)
Comment (Graywolf9) [reply files/es/learn/common_questions/web_mechanics/what_are_hyperlinks/index.md:15]: Honestly, I'm not pretty sure, there's no marks in `en-US`

Comment (JuanVqz) [reply files/es/learn/common_questions/web_mechanics/what_are_hyperlinks/index.md:15]: I know, the way it is structured in English makes me think it should has question marks but if you change it as you said will be better

Comment (Graywolf9) [reply files/es/learn/common_questions/web_mechanics/what_are_hyperlinks/index.md:15]: In Spanish the difference is in the accent in the word "como" or "cómo" and the obvious question marks, also how is it structured in English suggest that it is not a question, in that case it should says some like "how does the internet work?"

**Pull request merged**